test/integration/scheduler/podsready/scheduler_test.go

```diff
@@ -161,9 +155,10 @@ var _ = ginkgo.Describe("SchedulerWithWaitForPodsReady", func() {
 			ginkgo.By("checking the first prod workload gets admitted while the second is waiting")
 			prodWl := testing.MakeWorkload("prod-wl", ns.Name).Queue(prodQueue.Name).Request(corev1.ResourceCPU, "2").Obj()
 			gomega.Expect(k8sClient.Create(ctx, prodWl)).Should(gomega.Succeed())
+			util.ExpectWorkloadsToHaveQuotaReservation(ctx, k8sClient, prodClusterQ.Name, prodWl)
```

Comment:
I don't see how this change is related. Please move to a dedicated PR if not needed here.
.

Reply:
This and a couple of other small fixes ware added to solve some flakes observed after switching to cluster reuse.  

For this case, historically the test was passing due to a higher load at the beginning due to envtest initialization.
Now any of the two can get admitted first and by moving this line we make prodWl is admitted first.

Reply:
if these are fixes to flakes, then it makes sense to merge them separately so that we can have the release branch more stable? I see that it didn't fail before (almost never), but the point holds, IIUC, that the code was buggy.